### PR TITLE
FEAT: Update skin and icons

### PR DESCRIPTION
**Pull request by HenriqueLimas**
- Update to skin 16.8.0
- Update icons
- Update icon script logic to consider other elements other then `symbols`
- Change EbaySvg from display none to position absolute so `<defs>` are shown properly

https://opensource.ebay.com/ebayui-core-react/update-icons/index.html?path=/story/ebay-icon--all-icons

### CODE REVIEW

File: scripts/update-icons.js [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
-const skinSVGSymbols = parseSVG(skinIconsFile);
+const skinSVG = parseSVG(skinIconsFile)
 
-console.log(`Found ${skinSVGSymbols.length} icons in Skin.`)
-
-saveSvgIcons(skinSVGSymbols, ebaySvgFile)
+saveSvgIcons(skinSVG, ebaySvgFile)
 
+const skinSVGSymbols = parseSVGSymbols(skinIconsFile)
+console.log(`Found ${skinSVGSymbols.length} icons in Skin.`)
```
Comment (shpandian) [inline scripts/update-icons.js:18]: not part of this PR, but curious  - do we have any significance in this console log?

**Review by shpandian — APPROVED** [review]
LGTM

File: scripts/update-icons.js (hunk 1 above)
Comment (HenriqueLimas) [reply scripts/update-icons.js:18]: I think it just show as a feedback that the commend works
